frontend/src/locales/en-US.json

```diff
@@ -823,7 +823,11 @@
       "minimum-supported-engine-and-version": "{engine} >= {min_version} required",
       "restore-to-point-in-time": "Restore to point in time",
       "no-earlier-than": "Restore point-in-time cannot be earlier than [{earliest}] (the earliest available backup).",
-      "no-later-than-now": "Restore point-in-time cannot be later than now."
+      "no-later-than-now": "Restore point-in-time cannot be later than now.",
+      "target": "Restore to",
+      "target-inplace": "In place",
+      "target-new-db": "New database",
+      "will-override-current-data": "Will override current data"
```

Comment:
Suggest: "Will overwrite data in current database".

Reply:
SGTM